src/gpuhunt/providers/vultr.py

```diff
@@ -0,0 +1,181 @@
+    return RawCatalogItem(
+        instance_name=plan["id"],
+        location=location,
+        price=round(plan["monthly_cost"] / 730, 2),
+        cpu=plan["cpu_count"],
+        memory=plan["ram"] / 1024,
+        gpu_count=gpu_count,
+        gpu_name=gpu_name,
+        gpu_memory=gpu_memory,
+        gpu_vendor=gpu_vendor,
+        spot=False,
+        disk_size=plan["disk"],
+    )
+
+
+def get_instance_plans(plan: dict, location: str) -> Optional[RawCatalogItem]:
+    plan_type = plan["type"]
+    if plan_type in ["vc2", "vhf", "vhp", "voc"]:
+        return RawCatalogItem(
+            instance_name=plan["id"],
+            location=location,
+            price=plan["hourly_cost"],
+            cpu=plan["vcpu_count"],
+            memory=plan["ram"] / 1024,
+            gpu_count=0,
+            gpu_name=None,
+            gpu_memory=None,
+            gpu_vendor=None,
+            spot=False,
+            disk_size=plan["disk"],
+        )
+    elif plan_type == "vcg":
+        gpu_name = plan["gpu_type"].split("_")[1] if "_" in plan["gpu_type"] else None
+        if gpu_name in EXCLUSION_LIST:
+            logger.info(f"Excluding plan with GPU {gpu_name} as it is not supported.")
+            return None
+        gpu_vendor = get_gpu_vendor(gpu_name)
+        gpu_memory_gb = plan["gpu_vram_gb"]
+        gpu_count = (
+            max(1, gpu_memory_gb // get_gpu_memory(gpu_name, gpu_memory_gb)) if gpu_name else 0
+        )  # For fractional GPU,
+        # gpu_count=1
+        return RawCatalogItem(
+            instance_name=plan["id"],
+            location=location,
+            price=plan["hourly_cost"],
+            cpu=plan["vcpu_count"],
+            memory=plan["ram"] / 1024,
+            gpu_count=gpu_count,
+            gpu_name=gpu_name,
+            gpu_memory=gpu_memory_gb,
+            gpu_vendor=gpu_vendor,
+            spot=False,
+            disk_size=plan["disk"],
+        )
+
+
+def get_gpu_memory(gpu_name: str, memory: Optional[int] = None) -> float:
+    if memory:
+        for gpu in KNOWN_NVIDIA_GPUS:
+            if gpu.name == gpu_name.upper() and gpu.memory == memory:
+                return gpu.memory
+
+    for gpu in KNOWN_NVIDIA_GPUS:
+        if gpu.name == gpu_name.upper():
+            return gpu.memory
+
+    for gpu in KNOWN_AMD_GPUS:
+        if gpu.name == gpu_name.upper() and (memory is None or gpu.memory == memory):
+            return gpu.memory
+    logger.error(f"GPU {gpu_name} with memory {memory} not found in known GPU lists.")
+    raise ValueError(f"GPU {gpu_name} with memory {memory} not found.")
+
+
+def get_gpu_vendor(gpu_name: Optional[str]) -> Optional[str]:
+    if gpu_name is None:
+        return None
+    for gpu in KNOWN_NVIDIA_GPUS:
+        if gpu.name == gpu_name.upper():
+            return AcceleratorVendor.NVIDIA.value
+    for gpu in KNOWN_AMD_GPUS:
+        if gpu.name == gpu_name.upper():
+            return AcceleratorVendor.AMD.value
```

Comment:
(nit) Consider adding `.upper()` to `gpu.name`. Some GPUs have lowercase letters in the name.

Same thing in `get_gpu_memory`.

Reply:
Done